docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
+                "type": "integer"
+              },
+              "eventTime": {
+                "type": "date"
+              },
+              "eventTimeSource": {
+                "type": "keyword"
+              },
+              "eventTypeId": {
+                "type": "keyword"
+              },
+              "processId": {
+                "type": "keyword"
+              },
+              "threadId": {
+                "type": "keyword"
+              },
+              "userId": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      }
+    }
+
+The above JSON, returned from Elasticsearch, shows us the fixed and custom field type mappings for a tenant's index. An audit event's information is stored in fixed fields and the event's parameters are mapped to their appropriate types based on their field name suffixes added by the Audit library.
+
+A tenant application's audit events sent from the client-side library are added that tenant's index.
+
+    GET /audit_tenant_1/cafAuditEvent/_search
+    {
+      "took": 3,
+      "timed_out": false,
+      "_shards": {
+        "total": 5,
+        "successful": 5,
+        "failed": 0
+      },
+      "hits": {
+        "total": 2,
+        "max_score": 1,
+        "hits": [
+          {
+            "_index": "audit_tenant_1",
+            "_type": "cafAuditEvent",
+            "_id": "AVuvyhWuI0NChd-OZTz-",
+            "_score": 1,
+            "_source": {
+              "processId": "a040cdab-778d-4634-8b64-4fe4deedaa93",
+              "threadId": "1",
+              "eventOrder": "1",
+              "eventTime": "2016-11-15T14:30:00",
+              "eventTimeSource": "HOST1",
+              "userId": "JoeBloggs@yourcompany.com",
+              "correlationId": "correlation1",
+              "eventCategoryId": "documentEvents",
+              "eventTypeId": "deleteDocument",
+              "applicationId": "DocumentWebServiceApp",
+              "docId_CALng": "123456",
+              "authorisedBy_CAStr": "JoesphBloggins@yourcompany.com"
+            }
+          },
+          {
+            "_index": "audit_tenant_1",
+            "_type": "cafAuditEvent",
+            "_id": "AVuvySPNI0NChd-OZTzH",
+            "_score": 1,
+            "_source": {
+              "processId": "a040cdab-778d-4634-8b64-4fe4deedaa93",
+              "threadId": "1",
+              "eventOrder": "0",
+              "eventTime": "2016-11-15T14:12:12",
+              "eventTimeSource": "HOST1",
+              "userId": "JoeBloggs@yourcompany.com",
+              "correlationId": "correlation1",
+              "eventCategoryId": "documentEvents",
+              "eventTypeId": "viewDocument",
+              "applicationId": "DocumentWebServiceApp",
+              "docId_CALng": "123456"
+            }
+          }
+        ]
+      }
+    }
+
+The above JSON, returned from Elasticsearch, shows us all of the audit events belonging to a tenant.
```

Comment:
The above JSON, returned from Elasticsearch, displays all of the audit events belonging to "audit_tenant_1"